.gitignore

```diff
@@ -26,6 +26,7 @@ ChangeLog.rej
 Documents/
 Homepage/
 Test/
+!test/
```

Comment:
I added this because git was not seeing the new test file. Can be removed if it's a problem and test files can be manually added.

Reply:
`git add test/builtin/drawing/test_plot_compile.py` does not work?

Reply:
The question is if by adding this code, any test I have inside this folder is automatically added to the repo. Usually, I have some tests that I prefer not to include in a branch until the functionality they handle is completely supported. But it is not a big deal.